src/Maestro/Maestro.Web/Models/SubscriptionDependencyDetails.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using System.Linq;
+using Maestro.Data;
+using Maestro.Data.Models;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.HealthMetrics;
+
+namespace Maestro.Web
+{
+    public class SubscriptionDependencyDetails
+    {
+        public readonly string Repository;
```

Comment:
This should probably be get only auto-properties.  Public fields are against the design guidelines.